content/en/blog/_posts/2024-03-08-mid-cycle-1.30.md

```diff
@@ -0,0 +1,121 @@
+enables the creation of authorization chains with multiple webhooks with well-defined parameters
+that validate requests in a particular order and allows fine-grained control – such as explicit Deny
+on failures. The configuration file approach even allows you to specify
+[CEL](/docs/reference/using-api/cel/) rules to pre-filter requests before they are dispatched to
+webhooks, helping you to prevent unnecessary invocations. The API server also automatically reloads
+the authorizer chain when the configuration file is modified.
+
+You must specify the path to that authorization configuration using the `--authorization-config`
+command line argument. If you want to keep on using command line flags instead of using a
+configuration file, those will continue to work as-is. To gain access to new authorization webhook
+capabilities like multiple webhooks, failure policy, and pre-filter rules, switch to putting options
+in an `--authorization-config` file. From Kubernetes 1.30, the configuration file format is
+beta-level, and only requires specifying `--authorization-config` since the feature gate is enabled by
+default. An example configuration with all possible values is provided in the [Authorization
+docs](/docs/reference/access-authn-authz/authorization/#configuring-the-api-server-using-an-authorization-config-file).
+For more details, read the [Authorization
+docs](/docs/reference/access-authn-authz/authorization/#configuring-the-api-server-using-an-authorization-config-file).
+
+### Container resource based pod autoscaling ([KEP-1610](https://kep.k8s.io/1610))
+
+HPA ContainerResource type metric graduates to stable in v1.30. It allows you to configure
+Autoscaling based on resource usage of individual containers. See [our previous
+blog](/blog/2023/05/02/hpa-container-resource-metric/) for further details.
+
+### CEL for admission control ([KEP-3488](https://kep.k8s.io/3488))
+
+Integrating Common Expression Language (CEL) for admission control in Kubernetes introduces a more
+dynamic and expressive way of evaluating admission requests. This feature allows complex,
+fine-grained policies to be defined and enforced directly through the Kubernetes API, enhancing
+security and governance capabilities without compromising performance or flexibility.
+
+CEL's addition to Kubernetes admission control empowers cluster administrators to craft intricate
+rules that can evaluate the content of API requests against the desired state and policies of the
+cluster without resorting to Webhook-based access controllers. This level of control is crucial for
+maintaining the integrity, security, and efficiency of cluster operations, making Kubernetes
+environments more robust and adaptable to various use cases and requirements. For more information
+on the usage of CEL for admission control, see the [API
+documentation](/docs/reference/access-authn-authz/validating-admission-policy/) for
+ValidatingAdmissionPolicy.
```

Comment:
> Perhaps incorporating something similar to what was done in the past mid-cycle blog ([here](https://kubernetes.io/blog/2023/11/16/kubernetes-1-29-upcoming-changes/)).

@dipesh-rawat We originally had a similar section, but it was suggested to remove it since this blog is not focused on deprecations and removals.